### DISCUSSION TITLE
6.8 单源最短路径（二）

### DISCUSSION THREAD

**Discussion by giscus** [itcharge/AlgoNote#153] [Comments]
# 6.8 单源最短路径（二）

1. Bellman-Ford 算法 Bellman-Ford 算法：一种用于计算单源最短路径的算法，可以处理图中存在负权边的情况，并且能够检测负权环。

[https://algo.itcharge.cn/06_graph/06_08_graph_shortest_path_02/](https://algo.itcharge.cn/06_graph/06_08_graph_shortest_path_02/)

**Comment by xiangxichen26** [comment]
这里的内容咋都没了

**Reply by akxuan** [reply]
不如搞成开源的大家一起贡献

**Reply by itcharge** [reply]
不好意思久等了，文章内容已更新
